modules/org/infinispan/runtime/added/bin/launch.sh

```diff
@@ -108,28 +116,27 @@ fi
 
 # IDENTITIES_BATCH will not be set if IDENTITIES_PATH provided
 if [[ -n ${IDENTITIES_BATCH} ]]; then
-  # Use the native CLI if present
-  if [[ -f "${ISPN_HOME}/cli" ]]; then
-    ${ISPN_HOME}/cli --file ${IDENTITIES_BATCH}
-  else
-    ${ISPN_HOME}/bin/cli.sh --file ${IDENTITIES_BATCH}
-  fi
+  execute_cli --file "${IDENTITIES_BATCH}"
+fi
+
+if [[ -n ${SERVER_LIBS} ]]; then
+  execute_cli install --server-home="${ISPN_HOME}" "${SERVER_LIBS}"
 fi
 
 if [ -n "${DEBUG}" ]; then
-  tail -n +1 ${ISPN_HOME}/server/conf/*
+  tail -n +1 "${ISPN_HOME}"/server/conf/*
 fi
 
 # ===================================================================================
 # Server Execution
 # ===================================================================================
 
 if [ "$legacy" = true ]; then
-  exec ${ISPN_HOME}/bin/server.sh
+  exec "${ISPN_HOME}/bin/server.sh"
 else
   ARGS="$@"
   if [[ ! ${ARGS} =~ "--bind-address" ]]; then
     ARGS="--bind-address=0.0.0.0 ${ARGS}"
   fi
-  exec ${ISPN_HOME}/bin/server.sh ${ARGS}
+  exec "${ISPN_HOME}/bin/server.sh" "${ARGS}"
```

Comment:
Changes to the quotes here break the image. This is because of a bug in `server.sh`, the following `./server.sh "--bind-address=0.0.0.0 "` results in server startup failing with:

```java
2021-11-22 14:55:55,195 INFO  (main) [BOOT] JVM OpenJDK 64-Bit Server VM Red Hat, Inc. 11.0.13+8
2021-11-22 14:55:55,205 INFO  (main) [BOOT] JVM arguments = [-server, -Xlog:gc*:file=/tmp/infinispan-server-14.0.0-SNAPSHOT/server/log/gc.log:time,uptimemillis:filecount=5,filesize=3M, -Xms64m, -Xmx512m, -XX:MetaspaceSize=64M, -Djava.net.preferIPv4Stack=true, -Djava.awt.headless=true, -Dvisualvm.display.name=infinispan-server, -Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager, -Dinfinispan.server.home.path=/tmp/infinispan-server-14.0.0-SNAPSHOT, -classpath, :/tmp/infinispan-server-14.0.0-SNAPSHOT/boot/infinispan-server-runtime-14.0.0-SNAPSHOT-loader.jar, org.infinispan.server.loader.Loader, org.infinispan.server.Bootstrap, --bind-address=0.0.0.0 ]
2021-11-22 14:55:55,210 INFO  (main) [BOOT] PID = 268595
2021-11-22 14:55:55,235 INFO  (main) [org.infinispan.SERVER] ISPN080000: Infinispan Server starting
2021-11-22 14:55:55,235 INFO  (main) [org.infinispan.SERVER] ISPN080017: Server configuration: infinispan.xml
2021-11-22 14:55:55,236 INFO  (main) [org.infinispan.SERVER] ISPN080032: Logging configuration: /tmp/infinispan-server-14.0.0-SNAPSHOT/server/conf/log4j2.xml
2021-11-22 14:55:55,782 FATAL (main) [org.infinispan.SERVER] ISPN080028: Infinispan Server failed to start org.infinispan.commons.CacheConfigurationException: ISPN080033: Cannot find a network address which matches the supplied configuration
	at org.infinispan.server.configuration.ServerConfigurationParser.parseInterface(ServerConfigurationParser.java:209)
	at org.infinispan.server.configuration.ServerConfigurationParser.parseInterfaces(ServerConfigurationParser.java:196)
	at org.infinispan.server.configuration.ServerConfigurationParser.parseServerElements(ServerConfigurationParser.java:115)
	at org.infinispan.server.configuration.ServerConfigurationParser.readElement(ServerConfigurationParser.java:101)
	at org.infinispan.configuration.parsing.ParserRegistry.parseElement(ParserRegistry.java:204)
	at org.infinispan.configuration.parsing.ConfigurationBuilderHolder.handleAnyElement(ConfigurationBuilderHolder.java:160)
	at org.infinispan.commons.configuration.io.AbstractConfigurationReader.handleAny(AbstractConfigurationReader.java:57)
	at org.infinispan.configuration.parsing.Parser.readElement(Parser.java:98)
	at org.infinispan.configuration.parsing.ParserRegistry.parseElement(ParserRegistry.java:204)
	at org.infinispan.configuration.parsing.ParserRegistry.parse(ParserRegistry.java:182)
	at org.infinispan.configuration.parsing.ParserRegistry.parse(ParserRegistry.java:170)
	at org.infinispan.configuration.parsing.ParserRegistry.parse(ParserRegistry.java:164)
	at org.infinispan.server.Server.parseConfiguration(Server.java:306)
	at org.infinispan.server.Server.<init>(Server.java:224)
	at org.infinispan.server.Bootstrap.runInternal(Bootstrap.java:163)
	at org.infinispan.server.tool.Main.run(Main.java:98)
	at org.infinispan.server.Bootstrap.main(Bootstrap.java:50)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.infinispan.server.loader.Loader.run(Loader.java:106)
	at org.infinispan.server.loader.Loader.main(Loader.java:51)
	Suppressed: java.net.UnknownHostException: 0.0.0.0 : Name or service not known
		at java.base/java.net.Inet4AddressImpl.lookupAllHostAddr(Native Method)
		at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:929)
		at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1519)
		at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:848)
		at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1509)
		at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1368)
		at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1302)
		at java.base/java.net.InetAddress.getByName(InetAddress.java:1252)
		at org.infinispan.server.network.NetworkAddress.inetAddress(NetworkAddress.java:145)
		at org.infinispan.server.network.NetworkAddress.fromString(NetworkAddress.java:101)
		at org.infinispan.server.configuration.InterfaceConfigurationBuilder.createNetworkAddress(InterfaceConfigurationBuilder.java:43)
		at org.infinispan.server.configuration.InterfaceConfigurationBuilder.address(InterfaceConfigurationBuilder.java:23)
		at org.infinispan.server.configuration.ServerConfigurationParser.parseInterface(ServerConfigurationParser.java:218)
		... 22 more
```

Reply:
I've removed my changes